static/app/views/dashboardsV2/data.tsx

```diff
@@ -5,7 +5,7 @@ import {DashboardDetails} from './types';
 export const EMPTY_DASHBOARD: DashboardDetails = {
   id: '',
   dateCreated: '',
-  createdBy: '',
+  createdBy: undefined,
```

Comment:
should we also change `id` and `dateCreated` to undefined?

Reply:
I changed the type of createdBy -> User which is why I set it as undefined. the other two were string types- so I just left them as is since I didn't change the type